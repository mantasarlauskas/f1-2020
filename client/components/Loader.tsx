import React from 'react';
import styles from 'client/components/Loader.less';

function Loader() {
    return (
        <div className={styles.loader}>
            <div className={styles.circle} />
        </div>
    );
}

export default Loader;
