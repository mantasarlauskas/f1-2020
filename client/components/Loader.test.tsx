import React from 'react';
import Loader from 'client/components/Loader';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

describe('<Loader />', () => {
    it('matches snapshot', () => {
        const { asFragment } = render(<Loader />);
        expect(asFragment()).toMatchSnapshot();
    });
});
