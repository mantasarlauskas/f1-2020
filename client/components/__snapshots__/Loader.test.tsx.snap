// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="loader"
  >
    <div
      class="circle"
    />
  </div>
</DocumentFragment>
`;
